src/styles/utils/variables.scss

```
@@ -0,0 +1,8 @@
+$cell-size: 300px;
+$colors:
+  "1" red,
+  "2" rgb(204, 0, 0),
```

you can use percentages with `rgb()`
```suggestion
  "2" rgb(80%, 0, 0),
```